### DISCUSSION TITLE
ScriptReference/Playables.PlayableGraph.IsDone

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#653] [Notes]
# ScriptReference/Playables.PlayableGraph.IsDone

https://docs.unity3d.com/ScriptReference/Playables.PlayableGraph.IsDone.html

**Comment by radiatoryang** [comment]
Up until 2022.1 (and maybe later) this doesn't work like you'd expect, and it will return false even if the animation is done playing. See https://forum.unity.com/threads/animationclipplayable-stopping-behaviour.1130128/

To actually check doneness, try: 
```if( playableGraph.GetTime( ) > playableGraph.GetAnimationClip( ).length )```
